### PR TITLE
✨feat: Add getPadNumber util

### PR DESCRIPTION
**Pull request by SoYoung210**
---
name: ☄️ Pull Request Template
about: Pull Request! 
---
related issue : #5 

## 🤖Summary
* add getPadNumberUtil 

## Example
```js
getPadNumber(2, 1);
// '01'

getPadNumber(3, 1);
// '001'
```

### CODE REVIEW

**Comment by Brew-Brew** [comment]
I can't understand intuitively what situations can be used to use your utils . so can you add example in your issue or test code?

File: utils/spaceship.getPadNumber/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function getPadNumber(padCount, value) {
```
Comment (livelikeabel) [inline utils/spaceship.getPadNumber/index.js:1]: What kind of situation is the util used?🙋‍♂️
I don't know about exactly `PAD` mean.

File: utils/spaceship.uniqueKey/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function uniqueKey() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
```
Comment (livelikeabel) [inline utils/spaceship.uniqueKey/index.js:2]: Why is there a `4` as default string?✋

**Review by livelikeabel — COMMENTED** [review]
Plz check my comment 🙏

**Comment by SoYoung210** [comment]
> I can't understand intuitively what situations can be used to use your utils . so can you add example in your issue or test code?

It is a utility to match the number of digits. As you can see from the use case, it is 001 when inputting with padNumber (3,1).

File: utils/spaceship.uniqueKey/index.js (hunk 2 above)
Comment (SoYoung210) [reply utils/spaceship.uniqueKey/index.js:2]: It was an intermediate separator, but I do not think I would need it.
i'ill remove it

**Review by Brew-Brew — APPROVED** [review]

**Comment by SoYoung210** [comment]
@zx6658 I add isMobile util that wip. 

Can you check my test code? 🙏

**Comment by SoYoung210** [comment]
@zx6658 Thanks! i check [this comment](https://github.com/emplody/spaceship/pull/14#issuecomment-515702140) .

**Comment by Brew-Brew** [comment]
@SoYoung210 yeap, before i check for your commit, can you check my comments? https://github.com/emplody/spaceship/pull/14#issuecomment-515702140

I think you can find solution easily :)

**Pull request merged**